spec/models/story_spec.rb

```diff
@@ -1,5 +1,50 @@
 require 'rails_helper'
 
 RSpec.describe Story, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before :all do
+    @story = Story.create(title: 'Story one', content: 'Story content')
+  end
+
+  describe 'association' do
+    it { should have_many(:taggings) }
+    it { should have_many(:tags).through(:taggings) }
+    it { should belong_to(:user) }
+  end
+
+  describe 'search title' do
+    it 'should not find title' do
+      expect(Story.search('Story two')).to_not eql('Story one')
+    end
+  end
+
+  describe 'tag list' do
+    it 'should add tags' do
+      @story.tag_list = 'tag1, tag2, tag3'
+      expect(@story.tags).to match_array([
+                                           have_attributes(id: 1, name: 'tag1'),
+                                           have_attributes(id: 2, name: 'tag2'),
+                                           have_attributes(id: 3, name: 'tag3')
+                                         ])
+    end
+
+    it 'should skip some tags' do
+      @story.tag_list = 'tag1, tag2, tag3, tag4, tag5'
```

Comment:
which tags are skipped? If you think on ones that are added in previous scenario, I think the above scenario has nothing to do with this one, since they are on the same level - they execute independently.